README.md

```diff
@@ -52,9 +52,20 @@ git clone https://github.com/aziontech/cells-site-template
 
 ## Build
 
-You will need a bucket in AWS S3, its access key and its secret access key. You can set the credentials at the `azion.json` file inside the project or define the environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. The access credentials to the bucket will be stored in the code sent to Azion and the Edge Function will use them to retrieve the necessary pages stored during the publish step.
-Similarly, you will need credentials to deploy your project at Azion and you can either set those in the `azion.json` file or define the environment variable `AZION_TOKEN`. The Azion credentials are not stored in the built code.
+You will need a bucket in AWS S3, its access key and its secret access key. You can set the credentials at the `azion.json` file inside the project or define the environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
 
+You can alternatively enter the bucket data in the `environment variables`:
+
+```
+AWS_DEFAULT_BUCKET_NAME=bucket-name
+AWS_DEFAULT_BUCKET_REGION=sa-east-1
+AWS_DEFAULT_BUCKET_PATH=path
+```
+
+The access credentials to the bucket will be stored in the code sent to Azion and the Edge Function will use them to retrieve the necessary pages stored during the publish step.
+Similarly, you will need a [Personal Token](https://www.azion.com/pt-br/documentacao/produtos/gestao-de-contas/personal-tokens) to deploy your project at Azion and you can either set those in the `azion.json` file or define the environment variable `AZION_TOKEN`. The Azion credentials are not stored in the built code.
```

Comment:
You can export it as an environment varialbe, or use azioncli configure -t <token> command.

Reply:
azion-framework-adapter is not only used in cli context, it is also used in vscode plugin. All operations in the jamstack build/publish context must remain in this repository.